src/main/helm/helm-service.ts

```diff
@@ -95,21 +93,27 @@ class HelmService {
 
     return { message: output };
   }
+}
 
-  protected excludeDeprecated(entries: any) {
-    for (const key in entries) {
-      entries[key] = entries[key].filter((entry: any) => {
-        if (Array.isArray(entry)) {
-          return entry[0]["deprecated"] != true;
-        }
+class ChartGroups {
```

Comment:
This (imo) is not the sort of reason to introduce a class. This could easily just be a free function given it is immediately deconstructed after `new ChartGroups` call.

Reply:
Totally agree, but this was my experiment of making code easier to read and understand. `HelmChartGroups` seems like a separate entity to me, so I tried to move it into separate class along with related operations. If it's harder to read/understant, I can make a functions without a class.

Reply:
I comment the effort but I really don't think that classes should be used like this. It isn't an object that we are using merely we are using it for filtering.

Reply:
Replaced class with common methods.